### DISCUSSION TITLE
Expo app with oAuth UI

### DISCUSSION THREAD

**Discussion by nikolaEngIt** [supabase/supabase#12476] [Feature Requests]
So can we have a working example of an Expo app made with React Native(with JS, not TS)? And with implemented oAuth UI. Right now I am not able to do so. There is a problem with the versioning of React and "@supabase/auth-ui-react". Iven if I manage to install it the app won't start with error like : 
Unable to resolve "@stitches/core" from "node_modules\@supabase\auth-ui-react\dist\cjs\src\components\Auth\Auth.js"
In advance, I am sorry if this is the wrong category or I didn't give enough info.

**Comment by lucksp** [comment]
I am also seeing issues with React Native & the `Auth` component; related to svg `path`
